evap/evaluation/models.py

```diff
@@ -812,6 +1083,15 @@ class Meta:
     def is_general(self):
         return self.contributor_id is None
 
+    @property
+    def object_to_attach_logentries_to(self):
+        return Evaluation, self.evaluation_id
+
+    def __str__(self):
+        if self.contributor:
+            return _("Contribution by") + " " + self.contributor.full_name
+        return str(_("General Contribution"))
```

Comment:
Why the str()?

Reply:
![image](https://user-images.githubusercontent.com/5970076/91754890-7f4e9900-ebca-11ea-8916-21e687dfa50d.png)

Reply:
we're using gettext_lazy